views/tests.ejs

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <!--Copyright 2020 Google LLC
+
+    Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at
+
+         https://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.-->
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width">
+
+    <title><%= title %></title>
+
+    <style>
+      @media (prefers-color-scheme: dark) {
+        body {
+          background-color: #111;
+          color: white;
+        }
+      }
+    </style>
+  </head>
+  <body>
+    <p id="status">Running tests...</p>
+    
+    <script src="/resources/json3.min.js"></script>
```

Comment:
Can you add a TODO linking to https://github.com/foolip/mdn-bcd-collector/issues/407 here?